source/drivers/Makefile

```
@@ -0,0 +1,234 @@
+
+# `everything` builds the kernel base file and both standalone ROMs
+# across all six variant combinations (default, INVERT_SHIFT, INVERT_CTRL,
+# NO_UNDOC, NO_UNDOC+INVERT_SHIFT, NO_UNDOC+INVERT_CTRL). Each variant
+# runs as a recursive make invocation with the corresponding env vars
+# set. The default variant is built last so the on-disk intermediate
+# files are left in the non-inverted, undoc state.
+#
+# Each sub-make is prefixed with `-` and passes --keep-going so a
+# failure in one variant does not stop the other five from running.
+.PHONY: everything
+everything:
```

checkmake (0.3.2)

[warning] Target body for "everything" exceeds allowed length of 5 lines (6).

(maxbodylength)